docs/docs/guides/labs/components/managing-multiple-projects.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Managing multiple projects with Components'
+sidebar_position: 40
+---
+
+:::info
+
+This feature is still in development and might change in patch releases. It’s not production-ready, and the documentation may also evolve. Stay tuned for updates.
+
+:::
+
+You can use a _workspace_ directory to manage multiple projects within a single coherent directory structure.
+
+A workspace directory contains a root `pyproject.toml` containing workspace-level settings and a `projects` directory containing one or more projects. A workspace does _not_ define a Python environment by default. Instead, Python environments are defined per project.
+
+## Scaffold a new workspace
+
+To scaffold a new workspace with an initial project called `project-1`, run:
+
+<CliInvocationExample path="docs_beta_snippets/docs_beta_snippets/guides/components/workspace/1-dg-init.txt" />
+
+This will create a new directory called `workspace`, as well a new directory `project-1` within the `projects` folder. It will also setup a new uv-managed Python environment for the project. Let's look at the structure:
+
+<CliInvocationExample path="docs_beta_snippets/docs_beta_snippets/guides/components/workspace/2-tree.txt" />
+
+Importantly, the `pyproject.toml` file for the `workspace` folder contains an `is_workspace` setting marking this directory as a workspace:
+
+<CodeExample path="docs_beta_snippets/docs_beta_snippets/guides/components/workspace/3-pyproject.toml" language="TOML" title="workspace/pyproject.toml" />
+
+:::note
+
+`project-1` also contains a virtual environment directory `.venv` that is not shown above. This environment is managed by `uv` and its contents are specified in the `uv.lock` file.
+
+:::
+
+The `project-1` directory contains a `pyproject.toml` file that defines
+it as a Dagster project and component library:
```

Comment:
Let's just remove the component library bit.

Reply:
@schrockn what do you mean here exactly, removing it in the docs? or remove it from what gets scaffolded by default for a new project

Reply:
Just in the docs. I think it is confusing to frontload that concept. Component library should live in its own page IMO.